README.md

```diff
@@ -76,17 +76,19 @@ SDK Specific Options:
-| topOffset                 | How much space (px or %) to create between the top of the screen and messages. Not applicable for center, bottom, or full-screen messages                                                                                                | `string`                                                           | `undefined` |
-| closeButton               | Properties to show a custom close button on each in-app message                                                                                                | `CloseButtonOptions` (see below)                                                           | `undefined` |
+| Property Name                                    | Description                                                                                                                                                                                                                       | Value                                                             | Default     |
+| ------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------- | ----------- |
+| animationDuration                                | How much time (in MS) for messages to animate in and out                                                                                                                                                                          | `number`                                                          | `400`       |
+| bottomOffset                                     | How much space (px or %) to create between the bottom of the screen and messages. Not applicable for center, top, or full-screen messages                                                                                         | `string`                                                          | `undefined` |
+| displayInterval                                  | How much time (in MS) to wait before showing next in-app message after closing the currently opened one                                                                                                                           | `number`                                                          | `30000`     |
+| handleLinks<span style="color:#EF3D55">\*</span> | How to open links. If `undefined`, use browser-default behavior. `open-all-new-tab` opens all in new tab, `open-all-same-tab` opens all in same tab, `external-new-tab` opens only off-site links in new tab, otherwise same tab. | `'open-all-new-tab' \| 'open-all-same-tab' \| 'external-new-tab'` | `undefined` |
```

Comment:
This is the only line changed in this table. Added a red asterisk after handleLinks.